report/sections/lessons-learned-perspective/reflection.tex

```diff
@@ -0,0 +1,21 @@
+\subsection{Reflection}
+We believe we have achieved a high flow rate in our project, by fully automating the process from code to server to ``customer''.
+This has allowed us to work incrementally change our system, and get quick feedback from the customers and monitoring systems.
+Our automation systems also allows for easy rollback of any changes that may have caused issues.\\
+
+This way of deploying a lot instead of large deployments is very different from our usual projects. As they would usually be building the project and then releasing the entire thing at once.
```

Comment:
```suggestion
This way of deploying often instead of large deployments is very different from our usual projects. As they would usually be building the project and then releasing the entire thing at once.
```